src/consolelogger.c

```diff
@@ -221,10 +234,23 @@ void consolelogger_log(LOG_CATEGORY log_category, const char* file, const char*
     (void)vprintf(format, args);
     va_end(args);
 
+#if (defined(_MSC_VER) && defined(_DEBUG))
+    va_start(args, format);
```

Comment:
>va_start(args, format); [](start = 4, length = 23)

My varargs is rusty, but do we need to do the va_end() above here and another va_start here?  Can we just re-use the same args or is something happening under covers after initial vprintf() on 234 that requires this reset?